src/news-feed-events/NewsFeedEvent.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from "react";
+
+/**
+ * Represents an action taken.
+ */
+export default interface NewsFeedEvent {
+  "UNIQUE ID #": string;
+  "STATE": string;
+  "COUNTY": string;
+  "ACTION DATE": string;
+  // This column doesn't exist in the fake data.
+  "SOURCE"?: string;
+  "OVERALL BUCKET": string;
+  "SUB-CATEGORY": string;
+  "ACTION TAKEN": string;
+  "# OF PEOPLE RELEASED": string;
+  "# OF PEOPLE PREVENTED FROM ENTERING": string;
+  "SUMMARY RE-WRITE": string;
+}
+
+/**
+ * Convert a TSV spreadsheet into NewsFeedEvent objects.
+ */
+export function parseNewsFeedEventsFromTsv(tsv: string) {
+  let table = tsv.split("\n").map((line) => line.split("\t"));
+
+  const Headers = table[0] as (keyof NewsFeedEvent)[];
+  return table.slice(1).map((row) => {
+    let record = {} as NewsFeedEvent;
+    for (let i = 0; i < Headers.length; i++) {
+      let header = Headers[i];
+      let cell = row[i];
+      record[header] = cell;
+      1;
```

Comment:
Sorry for the ignorance: what is this line doing?

Reply:
No idea how that got there -- probably a typo. Thanks!